composer.json

```diff
@@ -167,6 +167,7 @@
         "drupal/libraries": "3.x-dev",
         "drupal/datalayer": "1.x-dev",
         "drupal/optimizely": "1.2",
+        "drupal/google_tag": "1.x-dev",
```

Comment:
Never
Please, please, please
NEVER add dependency development version of module
@Sanchiz @dave-the-brave 
Now OpenY is broken because maintainer of the module committed not finished code into dev branch